GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -47,6 +47,14 @@ public InProcessMountVerb()
             HelpText = "Show the debug window.  By default, all output is written to a log file and no debug window is shown.")]
         public bool ShowDebugWindow { get; set; }
 
+        [Option(
+            's',

```

Comment:
No need for a short name, since no user will be running this command

Reply:
(I know that is inconsistent with the other params here, but pretty much all of them are holdovers from the early days and should be removed anyways)